docs-v2/integrations/all/hubspot.mdx

```diff
@@ -3,29 +3,146 @@ title: HubSpot
 sidebarTitle: HubSpot
 ---
 
-API configuration: [`hubspot`](https://nango.dev/providers.yaml)
+## Overview
 
-## Features
+<CardGroup cols={3}>
+<Card title="Pre-built tooling" icon="screwdriver-wrench" href="#pre-built-tooling"/>
+<Card title="Pre-built use-cases" icon="square-check" href="#pre-built-use-cases"/>
+<Card title="Pre-requisites" icon="triangle-exclamation" href="#pre-requisites"/>
+<Card title="Setup guide" icon="rocket" href="#setup-guide"/>
+<Card title="Useful links" icon="circle-info" href="#useful-links"/>
+<Card title="API gotchas" icon="biohazard" href="#api-gotchas"/>
+</CardGroup>
 
-| Features | Status |
-| - | - |
-| [Auth (OAuth)](/guides/getting-started/authorize-an-api-from-your-app) | ✅ |
-| [Sync data](/guides/getting-started/read-from-an-api) | ✅ |
-| [Perform workflows](/guides/getting-started/write-to-an-api) | ✅ |
-| [Proxy requests](/guides/proxy-requests-to-an-api) | ✅ |
-| [Receive webhooks](/guides/receive-webhooks-from-an-api) | ✅ |
+## Pre-built tooling
 
-<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+<AccordionGroup>  
+  <Accordion title="✅ Authorization">
+    | Tools | Status | 
+    | - | - | 
+    | Pre-built authorization (OAuth 2) | ✅ |
+    | Credentials auto-refresh | ✅ |
+    | Authorization input validation | ✅ |
```

Comment:
Not 100% sure which feature you refer to, is it the automatic credentials validation?
If you also mean the additional parameters, I think we should mention them explicitly as a separate feature?

Suggestion:
Automatic credentials validation
Automatic auth-parameters validation